instance/tests/models/test_openedx_instance.py

```diff
@@ -157,6 +157,7 @@ def test_domain_url(self):
         self.assertEqual(instance.domain, internal_lms_domain)
         self.assertEqual(instance.lms_preview_domain, internal_lms_preview_domain)
         self.assertEqual(instance.studio_domain, internal_studio_domain)
+        self.assertEqual(instance.studio_domain_nginx_regex, r'~^(studio\-sample\.example\.org)$')
```

Comment:
@itsjeyd Inside character classes, it's sometimes necessary to escape `-` (usually you just put it as the first character of a character class, so it doesn't need escaping).

When escaping a full regular expression, the square brackets around the character class would be escaped, so the hyphen wouldn't need escaping.  I suspect they want the `re.escape()` to be generic enough to also escape _part_ of a regular expression.  Just guessing though, I didn't check the version history of that function.